service/controller/key/key.go

```diff
@@ -43,18 +43,33 @@ func ExtractOperators(comps []releasev1alpha1.ReleaseSpecComponent) []releasev1a
 	return operators
 }
 
+func ExtractAllOperators(releases releasev1alpha1.ReleaseList) map[string]releasev1alpha1.ReleaseSpecComponent {
+	var operators = make(map[string]releasev1alpha1.ReleaseSpecComponent)
+
+	for _, release := range releases.Items {
+		for _, component := range release.Spec.Components {
+			if IsOperator(component) && (operators[BuildAppName(component)] == releasev1alpha1.ReleaseSpecComponent{}) {
+				operators[BuildAppName(component)] = component
+			}
+		}
+	}
+	return operators
+}
+
+func IsOperator(component releasev1alpha1.ReleaseSpecComponent) bool {
```

Comment:
We should use this function in line 39 in the `ExtractOperators` function.

Reply:
That's why I added the function, I just forgot to do it.